src/charm.py

```diff
@@ -353,6 +403,41 @@ def _get_tls_config(self) -> dict:
             }
         }
 
+    def _get_tracing_config(self) -> dict:
+        """Return dictionary with opentelemetry configuration if available."""
+        # ref: https://doc.traefik.io/traefik/ma<woke forced me to add this>ster/observability/tracing/opentelemetry/
+        if not self._is_tracing_enabled():
+            logger.error("tracing not enabled")
```

Comment:
Without tempo, will we see an error in traefik logs every juju event?

Reply:
uh good point, I'll tune this down.